recipes/cyclus/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "1.4.0rc1" %}
```

Comment:
They don't in this particular case.  Cyclus hasn't had an officialm release in over a year and no previous released versions were relocatable.